ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1532,6 +1433,191 @@ private enum UnityLayers {
     }
   }
 
+  private void RenderGuidance(Vessel active_vessel) {
+    string vessel_guid = active_vessel.id.ToString();
+    if (plugin_.HasVessel(vessel_guid) &&
+        plugin_.FlightPlanExists(vessel_guid)) {
+      // Here the vessel is known to the plugin and has a flight plan.
+      //Duplicate.
```

Comment:
What is meant by duplicate here?

Badly formatted comment.

Reply:
Clarified.